app/src/main/java/org/jellyfin/androidtv/ui/preference/screen/PlaybackPreferencesScreen.kt

```diff
@@ -151,6 +152,47 @@ class PlaybackPreferencesScreen : OptionsFragment() {
 				bind(userPreferences, UserPreferences.subtitlesBackgroundEnabled)
 			}
 
+			@Suppress("MagicNumber")
+			colorList {
+				setTitle(R.string.lbl_subtitle_fg)
+				entries = mapOf(0xFFFFFFFF to context.getString(R.string.color_white),
+						0XFF000000 to context.getString(R.string.color_black),
+						0xFF7F7F7F to context.getString(R.string.color_darkgrey),
+						0xFFC80000 to context.getString(R.string.color_red),
+						0xFF00C800 to context.getString(R.string.color_green),
+						0xFF0000C8 to context.getString(R.string.color_blue),
+						0xFFEEDC00 to context.getString(R.string.color_yellow),
+						0xFFD60080 to context.getString(R.string.color_pink),
+						0xFF009FDA to context.getString(R.string.color_cyan))
+				bind(userPreferences, UserPreferences.subtitlesTextColor)
+			}
+
+			@Suppress("MagicNumber")
+			seekbar {
+				setTitle(R.string.lbl_subtitle_stroke)
+				min = 0
+				max = 24
+				increment = 2
+				bind(userPreferences, UserPreferences.subtitleStrokeSize)
+			}
+
+			@Suppress("MagicNumber")
+			seekbar {
+				setTitle(R.string.lbl_subtitle_position)
+				min = 0
+				max = 300
+				increment = 20
```

Comment:
an increment of 10 might be more useful than 20

Reply:
Not sure about this, while it may offer more steps, i once had a crash when changing this value too quickly, havent been able to replicate it since, but have also made this change